text/processing.py

```diff
@@ -202,3 +204,28 @@ def run(self):
             self.lines = int(length * self.ratio)
 
         self.sh("zcat -f {text_file} | tail -n {num_lines} | gzip > {out}")
+
+
+class DiffJob(Job):
+    """
+    Return the set difference of two text files, where one line is one element.
+    """
+
+    def __init__(self, file1, file2):
+        self.file1 = file1
+        self.file2 = file2
+
+        self.out_file = self.output_path("diff.txt")
+
+        self.rqmt = {"cpu": 1, "time": 1, "mem": 1}
```

Comment:
If we want to use this job on larger files, then maybe we can think about allowing gzipped outputs and custom rqmt.